go/engine/login_provision.go

```diff
@@ -133,6 +134,9 @@ func (e *loginProvision) Run(ctx *Context) error {
 
 	e.G().KeyfamilyChanged(e.arg.User.GetUID())
 
+	// check to make sure local files stored correctly
+	e.verifyLocalStorage()
```

Comment:
shouldn't we error out if this doesn't work?

Reply:
I wanted to start with more information. In case verifyLocalStorage is incorrect I didn't want to break provisioning. But we can go for it.